code/iaas/config-item/server/src/main/java/io/cattle/platform/configitem/context/data/metadata/common/ServiceMetaData.java

```diff
@@ -183,6 +181,16 @@ void populatePortsInfo(Service service, String serviceName) {
             this.expose.addAll((List<String>) exposeObj);
         }
     }
+    
+    void setServiceMetadata(Service service, String serviceName) {
+        if (serviceName.equals("ipsec")) {
```

Comment:
are we positive connectivity check svc will fetch it from ipsec metadata? from what I see in this PR, it's fetching it from self service (which is connectivity check svc)

Reply:
with this change - all 3 containers of `ipsec` get the information and it works, need to understand why

Reply:
It's fetching from self service, which gets printed as `ipsec`. 

Tested using `ipsec`, all 3 containers of `ipsec` - `ipsec`, `connectivity-check` and `router` will get the metadata info, and it works.